components/EndOfJourney.tsx

```diff
@@ -1,13 +1,99 @@
+export enum EndJourneyType {
+  NoBenefit,
+  UrgentCare,
+  AntiviralBenefit,
+  TooManyDays,
 }
 
-export const EndOfJourney: React.FC<EndJourneyType> = ({ title, content }) => {
-  return (
-    <div className='w-full p-6 bg-bcLightBlueBackground'>
-      {title && <p className='text-bcBlueLink font-bold'>{title}</p>}
-      {content && <div className='pt-2'>{content}</div>}
-    </div>
-  );
+interface EndOfJourneyProps {
+  journeyEnd: EndJourneyType | null;
+}
+
+export const EndOfJourney: React.FC<EndOfJourneyProps> = ({ journeyEnd }) => {
+  switch (journeyEnd) {
+    case EndJourneyType.NoBenefit:
+      return <NoBenefit />;
+    case EndJourneyType.UrgentCare:
+      return <UrgentCare />;
+    case EndJourneyType.AntiviralBenefit:
+      return <AntiviralBenefit />;
+    case EndJourneyType.TooManyDays:
+      return <TooManyDays />;
+    default:
+      return null;
+  }
+};
+
+const EndOfJourneyContainer: React.FC = ({ children }) => {
+  return <div className='w-full p-6 bg-bcLightBlueBackground rounded'>{children}</div>;
+};
+
+const EndOfJourneyTitle: React.FC = ({ children }) => {
+  return <p className='text-bcBlueLink font-bold mb-2'>{children}</p>;
 };
+
+const NoBenefit: React.FC = () => (
+  <EndOfJourneyContainer>
+    <EndOfJourneyTitle>
+      Based on your answer you would likely not benefit from antivirals for COVID-19 at this time.
+    </EndOfJourneyTitle>
+    <p>
+      These medications have been approved specifically for people experiencing mild-moderate
+      COVID-19 symptoms and require a positive test to be confirmed. As with most medications, there
+      can be side effects when they are not used for their intended purpose.If you are experiencing
+      symptoms and are at risk of more severe disease due to personal risk factors, you are
+      encouraged to get tested. For more information, go to &#8203;
+      <a
+        aria-label='bc cdc when to get a covid test'
+        className='font-bold'
+        href='http://www.bccdc.ca/health-info/diseases-conditions/covid-19/testing/when-to-get-a-covid-19-test'
+        target='_blank'
+        rel='noreferrer'
+      >
+        http://www.bccdc.ca/health-info/diseases-conditions/covid-19/testing/when-to-get-a-covid-19-test
+      </a>
+    </p>
+  </EndOfJourneyContainer>
+);
+
+const UrgentCare: React.FC = () => (
+  <EndOfJourneyContainer>
+    <EndOfJourneyTitle>
+      Please go to an urgent care clinic or emergency department as soon as possible.
+    </EndOfJourneyTitle>
+  </EndOfJourneyContainer>
+);
+
+const TooManyDays: React.FC = () => (
+  <EndOfJourneyContainer>
+    <p>
+      In order to be effective, the antiviral treatments that are currently available need to be
+      given within the first 5-7 days of symptom onset. Stay home and away from others until you
+      feel well enough to return to your regular activities. You should continue to seek medical
+      care if you feel you need it. For more information, go to&nbsp;
```

Comment:
&#8203; earlier, &nbsp; here. I may not fully know what 8203 does

Reply:
Ahh yeah this is a zero width space so we don't get a small space when the link causes a break 
![image](https://user-images.githubusercontent.com/71518072/151447003-fa8c7c9b-f82f-4278-bcf4-a1d2b9ec6948.png)
![image](https://user-images.githubusercontent.com/71518072/151447114-41aca795-d777-428e-b5d0-27bd8800732c.png)